vtex-apps/b2b-quotes
# [B2BORG-142] Hide for b2c

#### What problem is this solving?

This PR hide the link that direct to `b2b quote` for b2c users in users profile page

#### How to test it?

[Workspace](https://b2borg--sandboxusdev.myvtex.com/account#/profile)
Login with different users to check if the link hide for b2c users and show for b2b users.
